common/config/prometheus.go

```diff
@@ -38,6 +38,12 @@ func (p *Prometheus) ModifyConfig(dest ExporterConfigurer, currentConfig *Config
 
 	currentConfig.Exporters[rwExporterName] = GenericMap{
 		"endpoint": fmt.Sprintf("%s/api/v1/write", url),
+		"resource_to_telemetry_conversion": GenericMap{
+			"enabled": true,
+		},
```

Comment:
the resource can contain a lot of entries. are we sure this is safe? (will not blow up number of series)

did you consider using the [`PROMETHEUS_RESOURCE_ATTRIBUTES_LABELS`](https://docs.odigos.io/backends/prometheus) or add more values to the default of this field?